README.md

```diff
@@ -266,6 +269,22 @@ Examples:
 - `ref+s3://mybucket/mykey?region=us-west-2`
 - `ref+s3://mybucket/mykey?profile=prod`
 
+
+#### AWS KMS
+
+- `ref+awskms://PATH/TO/LOCAL/YAML/FILE.yaml?key_path=PATH.TO.KEY`
```

Comment:
If we were to name this `AWS KMS` provider, we'd better letting the user to write it like `ref+awskms://SOME_BASE64_ENCODED_CIPHER_STRING`, so that you can use the vals config file itself as the registry for keys and ciphertexts.

Reply:
If you need indirection to another YAML file external to the vals config file(or helmfile values.yaml), we'd better give it another, more "meta", name, like `vals` so that the usage looks like:

`ref+vals://path/to/another/config.yaml?key_path=PATH.TO.KEY`, so that you can virtually use any vals provider for indirection.

Reply:
Agree, but in the case with `vals` how we will understand that it is KMS encrypted secret value? Is it fine to add something like: 
`ref+vals://path/to/another/config.yaml?key_path=PATH.TO.KEY&encryption=kms` 

in that case we can add additional options later on.

Reply:
@mumoshu 

I changed the code and implement proposed solution, could you check it one more time please.